templates/organization/planning/_availabilities_assets.html.twig

```diff
@@ -0,0 +1,22 @@
+{% extends 'organization/planning/_availabilities_base.html.twig' %}
+
+{% block type %}{{ type }}{% endblock type %}
+
+{# columns number of blocks itemDataHeader and itemDataDetails should be the same than in _availabilities_users.html.twig #}
+{% block itemDataHeader %}
+    <th class="item-data">Radio mobile?</th>
+    <th class="item-data" colspan="2">Kit de premiers soins?</th>
+    <th class="item-data">Contact</th>
+    <th class="item-data" colspan="2">Lieu de stationnement</th>
+    <th class="item-data">Places</th>
+    <th class="item-data">Immatriculation</th>
```

Comment:
We should use translations here

Reply:
i'm waiting #289 to be merged

Reply:
Merge done, let me know if you want to address that in a separated PR